### PR TITLE
034-MicrosoftTeams-MakeItReal

### PR DESCRIPTION
**Pull request by edhild**
Contributing the Microsoft Teams Make It Real Challenge

### CODE REVIEW

File: 034-MicrosoftTeams-MakeItReal/Coach/01-collaboration.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Challenge 1: Collaboration Coach's Guide
+
+[< Previous Challenge](./00-prereqs.md) - **[Home](./readme.md)** - [Next Challenge>](./02-firstresponders.md)
+
+## Notes & Guidance
+Being the coaches guide, this content includes comments, guidance, possible gotchas related to the challenges facing the partipants. The success criteria for the challenge has been duplicated here, but within them, you will find the notes for coaches. In many cases, there is no single solution so coaches are encouraged to be flexibile and focus on the outcome. Did the team accomplish the spirit of the task not necessarily did they do it in the way it is described here?
+## Success Criteria
```
Comment (gfilicetti) [inline 034-MicrosoftTeams-MakeItReal/Coach/01-collaboration.md:7]: Add a blank line above this.

File: 034-MicrosoftTeams-MakeItReal/Student/01-collaboration.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## Learning Resources
+
+Reference articles on Microsoft Teams:
+- <https://docs.microsoft.com/en-us/MicrosoftTeams/get-started-with-teams-create-your-first-teams-and-channels>
+- <https://docs.microsoft.com/en-us/microsoftteams/teams-channels-overview>
+- <https://docs.microsoft.com/en-us/microsoftteams/quick-start-meetings-live-events>
+- <https://docs.microsoft.com/en-us/microsoftteams/teams-live-events/what-are-teams-live-events>
+- <https://docs.microsoft.com/en-us/microsoftteams/communicate-with-users-from-other-organizations>
+- <https://docs.microsoft.com/en-us/microsoftteams/built-in-custom-tabs>
+-<https://docs.microsoft.com/en-us/stream/overview>
```
Comment (gfilicetti) [inline 034-MicrosoftTeams-MakeItReal/Student/01-collaboration.md:42]: Need a space after the dash

File: 034-MicrosoftTeams-MakeItReal/Student/02-firstresponders.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+1. Demonstrate that a user can access the emergency information from a mobile device.
+1. Demonstrate that a user can take a set of documents from the emergency documentation offline on a device. 
+1. Demonstrate the ability to schedule and host a meeting where a user could dial in through telephony without the Microsoft Teams client. 
+1. Demonstrate how your team suggests standardizing the photo gathering business process of first responders. Based on the photo and information gathered, either the Logistics or Engineering team should be notified of the request. 
+
+## Advanced Challenges (Optional)
+
+*Too comfortable?  Eager to do more?  Try these additional challenges!*
+
+1. If the mobile device is not managed by the organization, the user is prompted for the multi-factor authentication.
```
Comment (gfilicetti) [inline 034-MicrosoftTeams-MakeItReal/Student/02-firstresponders.md:25]: Add pre amble sentence.

**Review by gfilicetti — CHANGES_REQUESTED** [review]
Great work, just a couple tiny things if  you agree.

**Review by gfilicetti — APPROVED** [review]
Awesome thank you for the fixes.

**Review by jrzyshr — APPROVED** [review]
Great work!  Thanks!

**Pull request merged**